soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -1057,6 +1289,227 @@ void DrawQuestStatusTab() {
     ImGui::PopItemWidth();
 }
 
+void DrawPlayerTab() {
+    if (gGlobalCtx) {
```

Comment:
Since this isn't a `bool` check:
```suggestion
    if (gGlobalCtx != nullptr) {
```